### PR TITLE
Hw 2 branch

### PR DESCRIPTION
**Pull request by ilkivs**
Homework 2 done
Created new branch for homework 2

### CODE REVIEW

File: src/HomeWork/hw2/Footballer.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    public double getTransferPrice() {
+        return transferPrice;
+    }
+
+    @Override
+    protected Object clone(){
+        try {
+            return super.clone();
+        } catch (CloneNotSupportedException e) {
```
Comment (Siril-O) [inline src/HomeWork/hw2/Footballer.java]: do not handle exception here, caller of clone should do this

File: src/HomeWork/hw2/Footballer.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            return new Footballer(getName(), getTeam(), getTransferPrice());
```
Comment (Siril-O) [inline src/HomeWork/hw2/Footballer.java]: do not call constructor

File: src/HomeWork/hw2/Footballer.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    public String getTeam() {
+        return team;
+    }
+
+    public double getTransferPrice() {
+        return transferPrice;
+    }
+
+    @Override
+    protected Object clone(){
```
Comment (Siril-O) [inline src/HomeWork/hw2/Footballer.java]: better return Footballer here

**Review by Siril-O — CHANGES_REQUESTED** [review]

File: src/HomeWork/hw2/Footballer.java (hunk 3 above)
Comment (ilkivs) [reply src/HomeWork/hw2/Footballer.java]: something like that?

    @Override
    protected Object clone() {
        return new Footballer(getName(), getTeam(), getTransferPrice());
    }

**Pull request merged**